Advanced-Recruit-Training/src/BatteryStateOfChargeService.cpp

```diff
@@ -8,6 +12,10 @@ namespace
 BatteryStateOfChargeService::BatteryStateOfChargeService(double initialStateOfChargePercent)
 : initialStateOfChargePercent_(initialStateOfChargePercent)
 {
+    initalAmpHour_= BATTERY_AMP_HOUR_CAPACITY*initialStateOfChargePercent_/100;
```

Comment:
this isn't quite what the AmpHour would be, think about what the initialstateofcharge represents